clang/test/Sema/warn-compare-enum-types-mismatch.c

```diff
@@ -1,12 +1,19 @@
 // RUN: %clang_cc1 -x c -fsyntax-only -verify -Wenum-compare -Wno-unused-comparison %s
 // RUN: %clang_cc1 -x c++ -fsyntax-only -verify -Wenum-compare -Wno-unused-comparison %s
 
+// In C enumerators (i.e enumeration constants) have type int (until C23). In order to support diagnostics such as 
```

Comment:
There's trailing whitespace here and it goes over the 80-col limit; can you reformat?

Reply:
done